packages/manager/apps/zimbra/src/pages/dashboard/Domains/Domains.tsx

```diff
@@ -57,13 +56,19 @@ const columns: DatagridColumn<DomainsItem>[] = [
 export default function Domains() {
   const { t } = useTranslation('domains');
   const { platformId } = usePlatform();
+  const { data } = useDomains();
+  const navigate = useNavigate();
   const { data: organization } = useOrganization();
-  const { data } = useQuery({
-    queryKey: getZimbraPlatformDomainsQueryKey(platformId, organization?.id),
-    queryFn: () => getZimbraPlatformDomains(platformId, organization?.id),
-  });
+  const shouldHideContent = useOverridePage();
   const handleAddDomain = () => {
-    console.log('Ajouter un nouveau domaine');
+    let urlWithParams = `${urls['add-domain'].replace(
+      ':serviceName',
+      platformId,
+    )}`;
+    if (organization?.id) {
+      urlWithParams += `?organizationId=${organization.id}`;
+    }
+    navigate(urlWithParams);
```

Comment:
Maybe we can use useHref
```suggestion
const hrefAddDomain = useHref(`./add-domain${organization?.id ? organizationId=organization.id : '' }`);
```